opensquirrel/decomposer/mckay_decomposer.py

```diff
@@ -23,6 +23,9 @@ def decompose(g: Gate) -> [Gate]:
         if abs(g.angle) < ATOL:
             return []
 
+        if g.name == "Rz" or g.name == "X90":
```

Comment:
Even though this decomposition is more efficient for many backends (including the qi backends), it is not McKay decomposition. If kept this way, then the docstring should be updated.